README.md

```diff
@@ -1,12 +1,12 @@
-Flink ML is a library which provides machine learning (ML) APIs and libraries that simplify the building of machine learning pipelines. It provides a set of standard ML APIs for MLlib developers to implement ML algorithms, as well as libraries of ML algorithms that can be used to build ML pipelines for both training and inference jobs.
+Flink ML is a library which provides machine learning (ML) APIs and a set of ML algorithms that simplifies the building of ML pipelines. Users can implement ML algorithms with the standard ML APIs and further use these algorithms to build ML pipelines for both training and inference jobs.
```

Comment:
The flink-ml-core provides not just API, but also `libraries that simplify the building of machine learning pipelines`.

Could you add it back?

Reply:
Well, when I was reading the README here, I find 
"Flink ML is a `library` which provides... and `libraries`"
 a bit confusing. 
How can a `library` provides `libraries`?

Reply:
Is there a way to add back the information?

Reply:
Thanks. I have rephrased it to 
`Flink ML is a library which provides machine learning (ML) APIs and infrastructures that simplifies the building of ML pipelines. Users can implement ML algorithms with the standard ML APIs and further use these infrastructures to build ML pipelines for both training and inference jobs.`